WordPressAuthenticator/Authenticator/WordPressAuthenticatorDisplayStrings.swift

```diff
@@ -79,7 +80,7 @@ public struct WordPressAuthenticatorDisplayStrings {
                 googleSignupInstructions: String = defaultStrings.googleSignupInstructions,
                 googlePasswordInstructions: String = defaultStrings.googlePasswordInstructions,
                 applePasswordInstructions: String = defaultStrings.applePasswordInstructions,
-                continueButtonTitle: String = defaultStrings.continueButtonTitle,
+                continueButtonTitle: String =  defaultStrings.continueButtonTitle,
```

Comment:
super nit: empty space on an unchanged line
```suggestion
                continueButtonTitle: String = defaultStrings.continueButtonTitle,
```